java/api/src/main/java/io/ray/api/runtimeenv/RuntimeEnvInfo.java

```diff
@@ -0,0 +1,23 @@
+package io.ray.api.runtimeenv;
+
+import io.ray.api.Ray;
+
+/** This is an experimental API to let you set runtime environments info for your actors. */
+public interface RuntimeEnvInfo {
```

Comment:
As discussed offline, I'm not sure if it's confusing to users vs. RuntineEnv

Reply:
We should remove this from user API.

Reply:
done.